travvy/src/store/store.js

```diff
@@ -0,0 +1,25 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
```

Comment:
would you say that using VueX for state management is the easist way to manage states in a Vue.js application?

Reply:
Yes, there is also a desktop extension for chrome which allows for easy state management tracking